### DISCUSSION TITLE
Which MATLAB 5G Tool box is used as baseline ?

### DISCUSSION THREAD

**Discussion by saiksaketh** [catkira/py3gpp#64] [General]
Hi, 
I have 2021a access, that 5G tool box is used as baseline ?

Because I think mathworks are evolving their releases!

How do the maintainers see this ?

Funny I ask this, as I left my old firm that has access to all matlab, and the new firm do not use any mathwork tools :D

**Comment by catkira** [comment]
I always use the latest version becaus I have an academic license. But it's a good point, we should specify for which Matlab version each test should pass.
@vborchsh Do you have an idea how to do this?

**Reply by saiksaketh** [reply]
Will be a good idea to assess freeze a certain versiion!

**Reply by vborchsh** [reply]
MATLAB has great back-compatibility between version. So next ten years we're happy, IMHO. But we can mention something like: "MATLAB 2022+ compatible"

**Reply by saiksaketh** [reply]
I have a 2021a, just let me know if thats relevant.
I can start the dev, and then see if the version issues pop up.

**Reply by catkira** [reply]
I think 2021a should be good as baseline. Can You try to run all the matlab tests an see if they pass?

**Reply by saiksaketh** [reply]
Thanks @catkira , Let me try this idea. if they all pass, safe to mark it as baseline!
I will update again

**Reply by saiksaketh** [reply]
Managed to get my hands on 2023a

I ran a matlab test

So looks I am good to go ?

`============================= test session starts =============================
collecting ... collected 6 items

test_nrCodeBlockDesegmentLDPC.py::test_nrCodeBlockDesegmentLDPC_fixed_data 
test_nrCodeBlockDesegmentLDPC.py::test_nrCodeBlockDesegmentLDPC[100] 
test_nrCodeBlockDesegmentLDPC.py::test_nrCodeBlockDesegmentLDPC[1000] 
test_nrCodeBlockDesegmentLDPC.py::test_nrCodeBlockDesegmentLDPC[4000] 
test_nrCodeBlockDesegmentLDPC.py::test_nrCodeBlockDesegmentLDPC[4001] 
test_nrCodeBlockDesegmentLDPC.py::test_nrCodeBlockDesegmentLDPC[10000] 

============================= 6 passed in 11.98s ==============================
PASSED [ 16%]PASSED [ 33%]PASSED [ 50%]PASSED [ 66%]PASSED [ 83%]PASSED [100%]
Process finished with exit code 0
`

**Reply by catkira** [reply]
yes
